indico/modules/events/editing/schemas.py

```diff
@@ -162,3 +162,24 @@ def _check_for_unique_tag_code(self, code):
             query = query.filter(EditingTag.id != tag.id)
         if query.has_rows():
             raise ValidationError(_('Tag code must be unique'))
+
+
+class EditableFileTypeArgs(mm.Schema):
+    class Meta:
+        rh_context = ('event', 'file_type')
+
+    name = fields.String(required=True, validate=not_empty)
+    extension = fields.List(required=True)
+    allow_multiple_files = fields.Boolean(required=True, validate=not_empty)
```

Comment:
you do not need those validates; most likely having those breaks the case of sending `false` because `not_empty` checks for `if not value`...